### ISSUE TITLE
Python 3.7 warning for EntryPoint

### ISSUE THREAD

**Issue by zuoxingdong** [openai/gym#1219]
In Python 3.7, it shows the following warning

```
gym/envs/registration.py:14: DeprecationWarning: Parameters to load are deprecated.  Call .resolve and .require separately.
    result = entry_point.load(False)
```
I think it needs to pass args when `require=False`

The related source code in pkg_resources is [here](https://github.com/pypa/setuptools/blob/master/pkg_resources/__init__.py#L2333-L2346)

**Comment by XinliZ** [comment]
Seeing same warning. Any solutions?
Env: windows 10.
Python 3.6.6

**Comment by AdamGleave** [comment]
This is fixed in PR #942, but has yet to be merged into the main branch.
